docs/source/contributing/rust-initiation.rst

```diff
@@ -0,0 +1,382 @@
+OpenDP Rust Initiation
+======================
```

Comment:
This notebook needs to be updated. Future issue.

Reply:
- Filed #1807